d2l-rubric-criteria-group.js

```diff
@@ -140,6 +140,9 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-rubric-criteria-group">
 				border-radius: 0.3rem;
 				border: 1px solid var(--d2l-color-celestine);
 			}
+			.criterion-cell.focused {
```

Comment:
```suggestion
			.criterion-cell:focus {
```

You can use the `:focus` CSS selector instead. Then the class and event handlers aren't necessary.